packages/playground/src/components/node_details_cards/card_details.vue

```diff
@@ -1,80 +1,92 @@
 <template>
-  <v-card :loading="loading">
-    <v-alert class="pa-5" style="height: 20px" v-if="title">
-      <h4 class="text-center font-weight-medium">
-        <v-icon :icon="icon" size="large" />
-        {{ title }}
-      </h4>
-    </v-alert>
-    <v-card-item :class="`mt-2 mb-2 ${$props.isMap ? 'map' : ''}`">
-      <v-alert type="error" variant="tonal" class="mt-2 mb-4" v-if="$props.error">
-        {{ $props.error }}
+  <v-card>
+    <template v-if="loading">
+      <v-skeleton-loader type="heading" class="ma-2" elevation="1" />
+      <v-skeleton-loader type="table-tbody" class="ma-2" elevation="1" />
```

Comment:
```suggestion
     <div class="pa-4">
        <v-skeleton-loader type="heading" class="mb-4" />
        <div v-for="n in 3" :key="n" class="d-flex justify-space-between align-center py-2 bb-gray">
          <v-skeleton-loader type="text" width="40%" />
          <v-skeleton-loader type="text" width="30%" />
        </div>
      </div>
```

![image](https://github.com/user-attachments/assets/3764e2f8-1330-4191-88ad-7eb5c16fcdc7)

Reply:
Close enough, good job!